WebLibs/api/IDAS.js

```diff
@@ -40,6 +42,19 @@ export class IDAS {
         }
     }
 
+    async authenticateWithJwt() {
+        if (!authJwtToken) {
+            const url = new URL(apiBaseUrl);
+            url.pathname = "/Session";
+            url.search = "?a=" + appToken;
+            url.search = url.search + "&r=%target%";
```

Comment:
There is no `a` and `r` in `SessionController.Index`

Reply:
I am reworking this code right now.
But SessionController.Index() for sure has these two parameters.

Reply:
Those parameters must be passed by the caller!

Reply:
Same mechanism as current SSO

Reply:
Sorry, I was looking at `POST Index`, not `GET Index` method. Everything is OK here